checks/raw/security_policy.go

```diff
@@ -27,17 +28,24 @@ import (
 	"github.com/ossf/scorecard/v4/log"
 )
 
+type SecurityPolicyFilesWithURI struct {
+	files []checker.File
+	uri   string
+}
+
 // SecurityPolicy checks for presence of security policy.
 func SecurityPolicy(c *checker.CheckRequest) (checker.SecurityPolicyData, error) {
-	files := make([]checker.File, 0)
-	err := fileparser.OnAllFilesDo(c.RepoClient, isSecurityPolicyFile, &files)
+	data := SecurityPolicyFilesWithURI{
+		files: make([]checker.File, 0),
+		uri:   "",
```

Comment:
Note: by default the string is the empty string, so no need for that

Reply:
Got it, thanks. For this one, should I keep it as it is now, or delete the empty string `uri` and commit again?

Reply:
it's fine like that it. One might argue it's more explicit. Let's merge in!

Reply:
No problem, thanks!